src/components/Tile/Tile.js

```diff
@@ -326,7 +326,10 @@ export class ExpandableTile extends Component {
         onClick={this.handleClick}
         tabIndex={tabIndex}>
         <button className="bx--tile__chevron">
-          <Icon name="chevron--down" description="Tile chevron" />
+          <Icon
+            name="chevron--down"
+            description={this.state.expanded ? 'Collapse' : 'Expand'}
```

Comment:
Taking this one step further, could we pass down something like `tileCollapseIconText` and `tileExpandIconText` as a prop, with 'Collapse' and 'Expand' as the defaults? Just thinking about future use-cases with regards to internationalization

Reply:
Good call.  I'll update to allow for that change.

Reply:
Thanks!